seirmo/tests/test_figures.py

```diff
@@ -0,0 +1,105 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import unittest
+from unittest.mock import patch
+
+import numpy as np
+import pandas as pd
+
+import seirmo as se
+
+
+class TestIncidenceNumberPlot(unittest.TestCase):
+    """
+    Test the 'IncidenceNumberPlot' class.
+    """
+    def test__init__(self):
+        se.IncidenceNumberPlot()
+
+    def test_add_data(self):
+        data = pd.DataFrame({'Time': [0, 1, 2, 3, 4, 5, 6],
+                                         'Incidence Number': [1, 2, 3, 4, 5, 6, 7]}) # noqa
```

Comment:
The line break here is a little weird, usually you would write
```
data = pd.DataFrame({
    'Time': [ .... ],
    'Incidence Number': [ ..... ]})
```
Then Flake8 won't complain and you adhere to the style standards.